core/controllers/base_test.py

```diff
@@ -288,6 +293,25 @@ def test_keys_match_en_qqq(self):
         qqq_key_list = self._extract_keys_from_json_file('qqq.json')
         self.assertEqual(en_key_list, qqq_key_list)
 
+    def test_keys_in_source_code_match_en(self):
+        """Tests that keys in HTML files are present in en.json."""
+        en_key_list = self._extract_keys_from_json_file('en.json')
+        for root, _, files in os.walk(os.path.join(
+                os.getcwd(), self.get_static_asset_filepath())):
+            for filename in files:
+                if filename.endswith('.html'):
+                    html_key_list = self._extract_keys_from_html_file(
```

Comment:
Can you also count the total number of HTML files checked, and then, at the very end of the function, add an assertion that this number is > 0? This is just some defense against changes to our file naming conventions in the future.